src/aker_metrics.c

```diff
@@ -0,0 +1,202 @@
+/**
+ * Copyright 2021 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <stdbool.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "aker_metrics.h"
+#include "aker_log.h"
+
+static aker_metrics_t *g_metrics = NULL;
+
+aker_metrics_t* get_global_metrics(void)
+{
+	aker_metrics_t* tmp = NULL;
+	tmp = g_metrics;
+	return tmp;
+}
+
+int init_global_metrics()
+{
+	aker_metrics_t* metrics;
+	metrics = (aker_metrics_t*)malloc(sizeof(aker_metrics_t));
```

Comment:
Drop the cast - that should not longer be a warning & is not the preferred pattern.

Use `calloc()` instead of `malloc()` and  you can also get rid of line 42-49.  Everything will be 0 safely.